recipes/alv/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "alv" %}
+{% set version = "1.5.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 53729576e59e3e99713d75fc43703c3170a115d951f00474f66b29a62062ac8e
+
+build:
+  number: 0
+  noarch: generic
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - biopython >=1.70
```

Comment:
I think only pip and python are needed in the host section

Reply:
I'll have to check this on Monday. Thanks for looking over the yaml!